lib/adding_columns.py

```diff
@@ -93,11 +94,30 @@ def add_average_films_ratings(spark: SparkSession, data: DataFrame) -> DataFrame
     data = data.join(data_with_ratings, on="nconst", how="left")
     return data
 
+def add_top_type(data: DataFrame) -> DataFrame:
```

Comment:
niestety wyniki ibydwu funkcji nie zgadzają się z tym jaki gatunek i typ najczęściej występuje :( sprawdziłam to na kilku linijkach w Excelu i nie zawsze wyrzucane są odpowieni typ i gatunek.

Reply:
sprawdziłam chwilowo dwie linijki i chwilowo się zgadza

Reply:
dobra, teraz znalazłam błąd dla aktora nconst='nm3716189'. Nie wiem czym to jest spowodowane..

Reply:
Poprawiłam kod w lib. Sprawdziłam na aktorze, który wcześniej się nie zgadzał. Teraz pokazuje dobrze go

Reply:
mi się nadal nie zgadza, zobacz przykład: tutaj ewidentinie widać, że dominuje titleType dominuje movie, a top_type jest short, natomiast dla genres dominuje Comedy, a top_genres jest Drama. 
![image](https://user-images.githubusercontent.com/116296531/214001167-58102561-f6be-47c5-99a5-542ef070a2af.png)

Reply:
a w tabelce pokazują się jeszcze inne wartości, więc mam wrażnie, że trochę to działa losowo 
![image](https://user-images.githubusercontent.com/116296531/214002309-33ca82a2-0ddd-4070-a8a0-43dab06e1a5d.png)